spec/controllers/api/admin/users/role_controller_spec.rb

```diff
@@ -105,10 +146,8 @@
       end
     end
 
-    context 'as conf_admin' do
```

Comment:
why not "as conf_admin" ?

as conf_admin do ... sounds pretty natural, doesn't it ?